config/gitleaks.toml

```diff
@@ -2406,6 +2406,14 @@ keywords = [
     "pscale_pw_",
 ]
 
+[[rules]]
+description = "Plivo Auth ID"
+id = "plivo-auth-id"
+regex = '''[SM]{1}A[0-9A-Za-z]{18}'''
```

Comment:
@mike-plivo this regex has a pretty good chance of generating a lot of false positives. Would you mind updating this to include a `semigeneric` prefix like this example https://github.com/gitleaks/gitleaks/blob/9c6650d14a6eb041444bceaefccdb76e527ecd0b/config/gitleaks.toml#L89

See how it is generated here: https://github.com/gitleaks/gitleaks/blob/9c6650d14a6eb041444bceaefccdb76e527ecd0b/cmd/generate/config/rules/alibaba.go#L30-L31